distribution/docs/src/main/resources/_contents/_configuring/configuring-from-the-admin-console-contents.adoc

```diff
@@ -0,0 +1,156 @@
+
+The ${branding} configuration can be performed through the ${admin-console} User Interface.
+
+==== Accessing the ${admin-console}
+
+Access the ${admin-console} from a web browser to configure the system.
+
+* Navigate to the ${admin-console}. (Default: \${secure_url}/admin)
+* Enter Username and Password. (Default: `admin/admin`)
+
+==== Initial Configuration
+
+The first time the ${branding} ${admin-console} runs, the initial configuration steps appear.
+
+. Click *Start* to begin.
+. On the next screen, general configuration settings such as host address, port and site name can all be configured. (See <<Configuring ${branding} Global Settings>> for important settings to configure)
+. Next, choose between a standard installation and a full installation. (Individual applications can be added, removed or deactivated later)
+
+[WARNING]
+====
+Platform App, Admin App, and Security Services App CANNOT be selected or unselected as it is installed by default and can cause errors if removed.
+====
+
+==== Viewing Currently Active Applications from ${admin-console}
+
+${branding} displays all active applications in the ${admin-console}.
+This view can be configured according to preference.
+Either view has an `>` arrow icon to view more information about the application as currently configured.
+
+.${admin-console} Views
+[cols="1,4", options="header"]
+|===
+|View
+|Description
+
+|Tile View
+|The first view presented is the Tile View, displaying all active applications as individual tiles.
+
+|List View
+|Optionally, active applications can be displayed in a list format by clicking the list view button.
+
+|===
+
+===== Application Detailed View
+
+Each individual application has a detailed view to view information specific to that application, adjust configurations or enable/disable features.
+All applications have a standard set of tabs, although some apps may have additional ones with further information.
+
+.Individual Application Views
+[cols="1,4", options="header"]
+|===
+|Configuration
+|The Configuration tab lists all bundles associated with the application as links to configure any configurable properties of that bundle.
+
+|Details
+|The Details tab gives a description, version, status, and list of other applications that either required for, or rely on, the current application.
+
+|Features
+|The features tab breaks down the individual features of the application that can be installed or uninstalled as configurable features.
```

Comment:
Do we explain how features can be started and stopped anywhere and how that affects the rest of the system?